Resources/views/CRUD/base_list_field.html.twig

```diff
@@ -11,14 +11,12 @@ file that was distributed with this source code.
 
 <td class="sonata-ba-list-field sonata-ba-list-field-{{ field_description.type }}" objectId="{{ admin.id(object) }}"{% if field_description.options.row_align is defined %} style="text-align:{{ field_description.options.row_align }}"{% endif %}>
     {% set route = field_description.options.route.name|default(null) %}
-    {% set action = route == 'show' ? 'VIEW' : route|upper %}
```

Comment:
I have an error in my project in sonata user admin (on this build):
Url: `http://127.0.0.1:8000/admin/sonata/user/user/list`
Error: `Variable "action" does not exist.` 
Row: `and admin.hasAccess(route, action in ['show', 'edit'] ? object : null)`
Symfony 2.8.18
User Bundle `3.x-dev a9a30bf`
Admin Bundle `3.x-dev 152beea`

After adding this removed line - everything works fine